recipe/meta.yaml

```diff
@@ -1,21 +1,19 @@
-{% set version = "4.2.2" %}
+{% set version = "5.2.0" %}
 
 package:
   name: bazel
   version: {{ version }}
 
 source:
   url: https://github.com/bazelbuild/bazel/releases/download/{{ version }}/bazel-{{ version }}-dist.zip
-  sha256: 9981d0d53a356c4e87962847750a97c9e8054e460854748006c80f0d7e2b2d33
+  sha256: 820a94dbb14071ed6d8c266cf0c080ecb265a5eea65307579489c4662c2d582a
   patches:
     - patches/0001-allow-args-to-be-passed-to-bazel_build.patch  # [unix]
     - patches/0002-do-not-use-enable-gold-unless-supported.patch  # [unix]
-    - patches/0003-Use-LSFindApplicationForInfo-for-locating-xcode.patch  # [osx]
     - patches/0004-link-against-iokit.patch  # [osx]
     - patches/0005-fix_netrc_protocol.patch  # [osx]
     - patches/0006-missing_inttype_h.patch
     - patches/0007-patch2.patch
-    - patches/0008-include-limits.patch
 
 build:
   number: 1
```

Comment:
Would we need to specify these deps?
https://github.com/bazelbuild/bazel/blob/master/MODULE.bazel#L12
I guess these are used? https://github.com/bazelbuild/bazel/tree/5.2.0/third_party

Reply:
My understanding is that Bazel is pretty much self-contained, once it is built, and it brings all its own dependencies in subprojects. Changing that would probably be a mammoth task.

Reply:
I don't think this patch was ever strictly needed.

Reply:
That makes sense, thank you.